src/main/java/jep/Jep.java

```diff
@@ -86,52 +86,13 @@
      * you make use of CPython extensions (e.g. numpy) that use the GIL, then
      * this gets even more risky and can potentially deadlock.
      */
-    private final static ThreadLocal<Boolean> threadUsed = new ThreadLocal<Boolean>() {
+    private static final ThreadLocal<Boolean> threadUsed = new ThreadLocal<Boolean>() {
```

Comment:
Can we use the diamond operator on the right side, i.e. `new ThreadLocal<>()`?

Reply:
The diamond operator would work here.